test/configs/zombieStarlight.json

```
@@ -46,15 +46,45 @@
     },
     "parachains": [
         {
-            "id": 2000,
+            "id": 1000,
             "chain_spec_path": "specs/single-container-tanssi-1000.json",
             "collators": [
                 {
-                    "name": "FullNode-2000",
-                    "command": "../target/release/container-chain-simple-node",
+                    "name": "FullNode-1000",
+                    "command": "../target/release/tanssi-node",
```

Now I see "dancebox/800" as the chain id in port 9948, this needs to be 
```suggestion
                    "command": "../target/release/container-chain-simple-node",
```

---

Also this is a full node, those should be defined outside of the `collators` array

---

no, I would not be able to create blocks if that is the case. Remember our node is not yet programmed to support starlight containbers, that is why I am directly putting dancebox as a container with their own collators